fabric/src/main/java/dev/schmarrn/lighty/fabric/ModMenuIntegration.java

```diff
@@ -0,0 +1,12 @@
+package dev.schmarrn.lighty.fabric;
+
+import com.terraformersmc.modmenu.api.ConfigScreenFactory;
+import com.terraformersmc.modmenu.api.ModMenuApi;
+import dev.schmarrn.lighty.ui.ModeSwitcherScreen;
+
+public class ModMenuIntegration implements ModMenuApi {
+    @Override
+    public ConfigScreenFactory<?> getModConfigScreenFactory() {
+        return ModeSwitcherScreen::new;
```

Comment:
It doesn't make sense to show the entire ModeSwitcherScreen, just showing the SettingsScreen is enough.

By showing the entire ModeSwitcherScreen, users can click on the "Overlay: OFF" Button, but unexpected things happen because there is no world loaded.

Reply:
I had thought this was by design, so that they can switch between the various modes. Wouldn't the SettingsScreen only bring up the screen with the slider? My experience with Lighty V2.2.0 is that F7 would bring up the "Lighty Mode Switcher" screen, and provide an option to bring up the SettingsScreen with the "Overlay Settings" button at the bottom.

I think being able to change Lighty modes via the settings screen is a good design choice, and should be kept. If you want users to start on the SettingsScreen, I think combining the various modes into one widget/button and moving it to the SettingsScreen would help to unify the screens into one without cluttering it. The new button would act like the existing "Graphics" or "GUI Scale" button, where clicking it would cycle between the various modes.